src/lsdb/loaders/dataframe/dataframe_catalog_loader.py

```diff
@@ -197,9 +197,7 @@ def _compute_pixel_list(self) -> list[HealpixPixel]:
             threshold=self.threshold,
             drop_empty_siblings=self.drop_empty_siblings,
         )
-        non_none_elements = alignment[alignment != np.array(None)]
-        pixel_list = np.unique(non_none_elements)
-        pixel_list = [HealpixPixel(order, pix) for (order, pix, count) in pixel_list if int(count) > 0]
+        pixel_list = list({HealpixPixel(tup[0], tup[1]) for tup in alignment if not tup is None})
```

Comment:
Here should we still make sure to filter by count, in case `drop_empty_siblings` is False?
```suggestion
        pixel_list = list({HealpixPixel(tup[0], tup[1]) for tup in alignment if not tup is None and int(tup[2]) > 0})
```

Reply:
No - those will be filtered out by the alignment logic and they will just be `None` instead of a tuple with a 0-value for the third element.